server_fn_macro/Cargo.toml

```diff
@@ -21,6 +21,8 @@ nightly = []
 ssr = []
 actix = []
 axum = []
+wasi = ["generic"]
```

Comment:
Once `wasm32-wasip2` is a stable target in Rust (coming in 1.82 afaik) the use of feature flags could be simplified, using the target directive instead https://doc.rust-lang.org/nightly/rustc/platform-support/wasm32-wasip2.html#conditionally-compiling-code

Reply:
True! I initially went for the feature to kind of follow the other integrations but since we have a target I may just use that!

Reply:
edit: This is live now in Rust 1.82!